en/advanced/rtk_gps.md

```diff
@@ -21,7 +21,7 @@ The datalink should typically be able to handle an uplink rate of 300 bytes per
 
 ## Supported RTK GPS modules
 
-PX4 currently only supports the single-frequency (L1) u-blox M8P based GNSS receivers for RTK.
+PX4 currently only supports the single-frequency (L1) u-blox M8P based GNSS receivers for RTK. <!-- F9P? -->
```

Comment:
@bkueng I do not believe this section (in the developer docs) is true. We support F9P too now! Can you provide a new comment about our level of support - ie what modules, what frequencies?

Also a quick scan of the rest of this doc for anything else out of date? Feel free to push in any changes to the branch.

Reply:
Thank you!